src/components/timecard/edit-shift-hours/EditShiftHours.spec.jsx

```diff
@@ -47,6 +55,28 @@ describe('EditShiftHours', () => {
     });
   });
 
+  it('should call updateTimeEntry when pressing save when there is an existing time entry', async () => {
+    renderWithTimecardContext(
+      <EditShiftHours
+        setShowEditShiftHours={jest.fn()}
+        timeEntry={{ ...newTimeEntry, timeEntryId: '1' }}
+        index={0}
+      />
+    );
+
+    act(() => {
+      const startTimeInput = screen.getByTestId('shift-start-time');
+      fireEvent.change(startTimeInput, { target: { value: '08:00' } });
+
+      const saveButton = screen.getByText('Save');
+      fireEvent.click(saveButton);
+    });
+
+    await waitFor(() => {
+      expect(mockUpdateTimeEntry).toHaveBeenCalled();
```

Comment:
Maybe check what this function is called with too?

Reply:
updated